DSharpPlus/Net/Rest/RateLimitStrategy.cs

```diff
@@ -59,15 +70,16 @@ TState state
 
         if (!exemptFromGlobalLimit && !this.globalBucket.CheckNextRequest())
         {
-            return this.SynthesizeInternalResponse(route, globalBucket.Reset, "global");
+            return this.SynthesizeInternalResponse(route, globalBucket.Reset, "global", traceId);
         }
 
         if (!this.routeHashes.TryGetValue(route, out string? hash))
         {
             logger.LogTrace
             (
                 LoggerEvents.RatelimitDiag,
-                "Route has no known hash: {Route}.",
+                "Request ID:{TraceId}: Route has no known hash: {Route}.",
```

Comment:
It should be `Request {TraceId}` imo

Reply:
discuss that with aki :when:

Reply:
i specifically requested this, easier to filter lines by ID: and get mostly relevant results

Reply:
Fair point